api/src/main/resources/static/js/table-enhancements.js

```diff
@@ -0,0 +1,136 @@
+// Functionality for dynamically resizeable table
```

Comment:
I haven't spun up the dictionary service to check this out, but it's impressive that you wrote your own resizing code.  While reviewing this ticket, did you take a look at any plugins or built-in jquery/datatables functionality which could be used to achieve similar results?

Doing a quick google, it looks like there might have been easier ways of getting resizing to work.
https://github.com/jeffreydwalter/ColReorderWithResize
https://stackoverflow.com/a/68012965

What are your thoughts on those?

Reply:
Thanks! I'll look into those and potentially change what I have. 
I followed a tutorial [link](https://brainbell.com/javascript/making-resizable-table-js.html) with only minor changes to how they did it. I'm not sure if this is an issue or how to go about citing that the bulk of the functionality came from here if I do end up keeping the code. I certainly wouldn't want to take credit for this js code.